src/test/java/uk/gov/dhsc/htbhf/utils/NoopWireMockManager.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.dhsc.htbhf.utils;
+
+/**
+ * Implementation of WireMockManager that does nothing which can be used for BrowserStack testing
+ * without having to have conditional logic in the steps.
+ */
+public class NoopWireMockManager implements WireMockManager {
```

Comment:
If this isn't doing anything then we can just have an interface with default implementation.

Reply:
Done.